### ISSUE TITLE
Strategic Intent and Tactical intent are not defined

### ISSUE THREAD

**Issue by inverted-capital** [dreamcatcher-tech/dreamcatcher-tech.github.io#26]
These sections need definition in the [Request Template](https://github.com/dreamcatcher-tech/dreamcatcher-tech.github.io/blob/master/website/nfas/Requests/00%20Request%20Template.md) and a difference between them should be stated.

Why is this not done using nested Requests ?  I agree that it should not be, but why ?  Is it because Strategic is vague, whereas Tactical is precise ?

**Comment by TemperBead** [comment]
Will transfer the below to a blog draft so I can pick this up, as just hammering it out cold, but in answer to your question:

It's not a vague vs precise thing.  It's headmark vs compass bearing thing.

It's from hours of having it drilled into me at naval college.  They summed it up as the difference between Ratcatchers and Regulators, and why you needed a middle way.  (BTW, I should probably knock out a blog post on this as it's still welded to my head and it speaks to what we're trying to do.  Will add it to the work stack.)

The general rule is that you should always know what your '1-up' is trying to do, regardless is what he's telling you to do.  It's recognising that centralised command and control is inferior to decentralised command and control, but by decentralising, you could give misfires.  You need to let the guys know not only what you think they need  to do, but why you've asked them to do it.  

So the strategic intent is the high level "Start with why".  That's needed because it's not possible to write the tactical intent (the compass bearing) in sufficient detail to avoid fuck ups by itself, you need to give a headmark.  But it's not enough to just state why you're trying to do something (the headmark), without giving some guidance.  

The Ratcatchers and Regulators thing comes from Admiral Beatty and Admiral Jellicoe in WW1.  Beatty was the ratcatcher, and didn't plan ahead at all well, and gave full reign to the captains under his command.  Didn't go well, as they each had a different idea of what the overall objective was, and so perfectly executed their own commands, just in different directions.  Jellicoe was the Regulator, and literally had a huge book of rules which all the captains under his command had to follow, and so was exceptionally brittle as soon as the hun came over the horizon.

The middle ground is the way.  Don't over specify, don't leave people in doubt of what the whole point of the thing is.

So it's fundamentally different from nested Requests, in that strategy is about why, then there's a phase change to tactical which is about how, what, where etc.

I'll get a definition into the template, but be warned, could talk for hours on this subject ;-)

**Comment by inverted-capital** [comment]
So Strategy is "Why we are making this Request" and Tactical is "What is this Request asking for" ?

I think this ticket closes when the blogpost is published and the template has a compressed version of the difference, yeah ?

**Comment by TemperBead** [comment]
Agreed.

**Comment by inverted-capital** [comment]
I do not think it is valid to close an issue until it is actually resolved - "I made a note to do this task somewhere else" isn't a closeable reason.  This ticket closes when either that blog post is complete, or the same information is placed somewhere else relevant, such as the Request Template.
